### DISCUSSION TITLE
Impianto irrigazione anti gatto

### DISCUSSION THREAD

**Discussion by dcamb** [indomus/forum#39] [Hardware & Firmware]
Ciao a tutti,
Sto automatizzando l'impianto di irrigazione 2 zone:
2 elettrovalvole 48V, alimentatore, shelly 2.5 wifi, Hassio su rspb.
Vorrei aggiungere 2 sensori pir da esterno, wifi, per azionare l'impianto al passaggio dei gatti, che vengono a "liberarsi" nel mio prato :-)
Non trovo dei sensori pir con sensibilità regolabile da esterno, wifi. Devono poter stare sotto la pioggia... 
Qualcuno ha dei suggerimenti? 
Grazie Davide

**Comment by KC-inDomus** [comment]
diabolico 😂

**Comment by Zatarraz** [comment]
Non saprei come fare, ma potresti mettere un telecamera da esterno con motion detector. Poca spesa e tanta resa

**Reply by dcamb** [reply]
Mi sembra un ottima idea, cerco una ezviz da integrare, pubblicherò gli screen del "lavaggio gatti" grazie!!
Davide

**Reply by KC-inDomus** [reply]
volendo far le cose per bene potresti usare ip cam + ternsorflow per il riconoscimento dei gatti dal flusso video.

**Reply by Nik71git** [reply]
Ho anch'io lo stesso problema e adotto la stessa soluzione, però manuale... Se dovessi settare le videocamere su motioneye per rilevare anche i gatti (il gatto !!! ...quello del vicino) sarei pieno di falsi allarmi, oltre al fatto che se vai in giardino attivi tu il trigger e fai scattare l'irrigazione.... Però la soluzione è intelligente e soprattutto legale ed incruenta. Ci sarebbe un'altra soluzione, identica come principio e sono quegli aggeggi con pannellino solare che pianti nel terreno, hanno un serbatoio per l'acqua ed un PIR ma ho seri dubbi sul funzionamento in quanto il gatto impara presto ad aggirarli. Infine ci sarebbe LA soluzione, ma non posso postare il link, un po' meno incruenta... si tratta di domotizzare una carabina ad aria compressa....

**Reply by Zatarraz** [reply]
Questa sarebbe LA SOLUZIONE DEFINITIVA, ma non è colpa dei gatti, quanto dei padroni...

**Reply by Nik71git** [reply]
La carabina domotizzata può servire anche per i padroni 😎
